oshc/oshc/settings.py

```diff
@@ -159,11 +159,12 @@
 
 STATICFILES_DIRS = (os.path.join(BASE_DIR, 'main/'), )
 
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_HOST_USER = 'email'
-EMAIL_HOST_PASSWORD = 'password'
-EMAIL_PORT = 587
+EMAIL_HOST = os.getenv('EMAIL_HOST', '')
+EMAIL_HOST_USER = os.getenv('EMAIL_ID', '')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_PASSWORD', '')
+EMAIL_PORT = os.getenv('EMAIL_PORT', '')
 EMAIL_USE_TLS = True
+
```

Comment:
Register/Login is of no good use if we don't confirm the email. Currently we have it hidden on master, once we are up with check we would make it live. This is needed and cannot be removed. Did you get the chance to set it on Heroku? or does this not work for you? I'm reopening the issues.

Reply:
@tapasweni-pathak There is a known issue with Gmail and Heroku. Heroku servers are located all around the world. When we try to send an email via Heroku server, Gmail blocks the request considering the unusual origin of the request. We could try and see if this works, but I'm not sure if this `email verification via emails from gmail` should be used on production.

Reply:
This happens with mutt to if you use it with Gmail, [this](https://support.google.com/accounts/answer/6010255?hl=en) needs to be set up in that case and we can restrict to heroku. Can you please drop a blog or something, as I'm not aware about [this](https://blog.heroku.com/tech_sending_email_with_gmail) but not about the known issue, thanks!